ui/redux/selectors.js

```diff
@@ -20,6 +20,7 @@ export const getVariantFunctionalDataByGuid = state => state.variantFunctionalDa
 export const getMmeSubmissionsByGuid = state => state.mmeSubmissionsByGuid
 export const getMmeResultsByGuid = state => state.mmeResultsByGuid
 export const getGenesById = state => state.genesById
+export const getPaGenesById = state => state.pagenesById
```

Comment:
you don't seem to use this data anywhere, or the `paLocusList` data on the locus list itself. Is there a separate PR where you show this information? If not, it would be way more performant not to fetch/ return the data to the client

Reply:
See answer below regarding UI.